crt_portal/static/sass/custom/form.scss

```diff
@@ -6,7 +6,7 @@
 form#report-form {
   .question_primary_complaint {
     ul {
-      padding-inline-start: 0;
+      padding-left: 0;
```

Comment:
`padding-inline` is super bleeding edge and has no support in any version of IE, so I replaced it with `padding-left`